.storybook/preview.js

```diff
@@ -6,7 +6,7 @@ import { GlobalStyle } from '../src/components/layout/global-style';
 addDecorator((storyFn) => (
   <>
     <ThemeProvider theme={theme}>
-      <GlobalStyle />
+      <GlobalStyle $lightTheme />
```

Comment:
This still led to the dark theme when I tried it:
![image](https://user-images.githubusercontent.com/7970458/141331934-d598c232-33a3-471f-a5e6-23227eae99b7.png)

We need to explicitly set this to `true` to make it work.
```suggestion
      <GlobalStyle $lightTheme={true} />
```